text-editor/text-editor-002-en.md

```diff
@@ -556,7 +556,84 @@ spend some time on one of these buttons : the tweet insertion.
 
 ## Clipboard access
 
-TBW
+Using `contentEditable` tags rather than regular form elements such as `input`
+has drawbacks. Say that our end users need to paste text from other sources to
+write articles. Doing so in an input element, everything works just great. You
+get rid of non textual elements, and end up with a raw string ready to be
+submitted. When pasting in contentEditable element, things are way different:
+The browser will try to render the pasted text as close as possible from of the
+original copied text, resulting in massive and ugly inline style everywhere.
+This completely messed up our editor because it violated the sanitization logic
+it's based on.
+
+We had to find a way to sanitize (once more !) the user clipboard, to ensure
+only raw text is pasted from external sources. Problem is, clipboard interaction
+is one of the most obvious security breach in web application, because it
+implies a direct communication between javascript and the external world with
+almost no way to control what's comming from it. In the past, several workaround
+involving extreme methods such as `iframe` and even `flash` (brrrr) were used.
+Lately, the well known `document.execCommand('paste')` came to the rescue, but
+was still very limitated due to security concerns.
```

Comment:
maybe add a link to some resources about that ? such as browsers compatibility table